src/modules/patient-detail-list/patient-list-filters/index.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { Button, Menu, Dropdown, Checkbox } from 'antd';
+import { useHistory, useLocation } from 'react-router-dom';
+import qs from 'qs';
+import { DownOutlined } from '@ant-design/icons';
+
+function PatientListFilters() {
+  const history = useHistory();
+  const location = useLocation();
+  const filterConfig = qs.parse(location.search, { ignoreQueryPrefix: true });
+
+  function onFilterSelect(type) {
+    console.log(type)
+    return value => {
+      history.push({
+        pathname: location.pathname,
+        search: qs.stringify(
+          Object.assign({}, filterConfig, {
+            [type]: value
+          })
+        )
+      });
+    };
+  }
+
+  const ageMenu = (
```

Comment:
use select instead of menu as done in facility list filters

Reply:
but in zeplin UI there are checkbox used to filter the data
https://app.zeplin.io/project/5e9ec53c2a1dc852e6ed14c6/screen/5eaba12d930bdeb102c4fae8

Reply:
yes, we will create a new component for that, this should not be done using menu, for now, use select as done in facility list filters